winui/Cartesian-Charts/Axis/Axislabels.md

```diff
@@ -13,7 +13,7 @@ Axis labels are used to show the units, measures, or category value of the axis
 
 ## Rotation Angle
 
-The [LabelRotationAngle](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html#Syncfusion_UI_Xaml_Charts_ChartAxis_LabelRotationAngle) property is used to define the angle for the label content.
+The [LabelRotation](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html#Syncfusion_UI_Xaml_Charts_ChartAxis_LabelRotationAngle) property is used to define the angle for the label content.
 
```

Comment:
@naveenkumar-sanjeevirayan  : Remove the old links

Reply:
@MuneeshKumarG , I have removed the old website links.